lib/rucio/common/dumper/consistency.py

```diff
@@ -41,10 +41,12 @@ def dump(cls, subcommand, ddm_endpoint, storage_dump, prev_date_fname=None, next
                 ddm_endpoint, prev_date, cache_dir=cache_dir)
             next_date_fname = data_models.Replica.download(
                 ddm_endpoint, next_date, cache_dir=cache_dir)
-            assert prev_date_fname is not None
-            assert next_date_fname is not None
+            if (prev_date_fname is None) or (next_date_fname is None):
+                raise ValueError("Both prev_data_fname and next_date_fname are required for subcommand='consistency'")
```

Comment:
Could be worth mentioning which one of the two is `None` (or if both are) in the exception message. Maybe it could either be:

```suggestion
            if prev_date_fname is None):
                raise ValueError("prev_data_fname is required for subcommand='consistency'")
            elif next_date_fname is None):
                raise ValueError("next_date_fname is required for subcommand='consistency'")
```

or keeping it in a single check and logging both values